src/tests/compiler/class/classDeclarationTests.ts

```diff
@@ -1347,4 +1348,64 @@ class Child extends Mixin(Base) {}
             doTest("class I {}\n\nclass J {}\n\nclass K {}", 1, "class I {}\n\nclass K {}");
         });
     });
+
+    describe(nameof<ClassDeclaration>(d => d.getStructure), () => {
+        function doTest(code: string, expectedStructure?: ClassDeclarationStructure) {
```

Comment:
Use `MakeRequired<ClassDeclaration>` here to ensure we test all the properties of `ClassDeclarationStructure`.